extensions/upcoming-holidays/package.json

```diff
@@ -2,43 +2,67 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "upcoming-holidays",
   "title": "Upcoming Holidays",
-  "description": "Show upcoming holidays for a country",
+  "description": "Show upcoming holidays for a country or region",
   "icon": "calendar-icon.png",
   "author": "rodrigouroz",
+  "contributors": [
+    "YourMCGeek"
+  ],
   "categories": [
     "Data",
     "Productivity"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "upcoming-holidays",
       "title": "List Upcoming Holidays",
-      "description": "Allows you to see the upcoming holidays for a country",
-      "mode": "view"
+      "description": "Allows you to see the upcoming holidays for a country and region (if applicable)",
+      "mode": "view",
+      "subtitle": "Upcoming Holidays"
+    },
+    {
+      "name": "all-holidays",
+      "title": "List All Holidays",
+      "description": "Allows you to see all the holidays, past and future, for a country and region (if applicable)",
+      "mode": "view",
+      "subtitle": "Upcoming Holidays"
+    },
+    {
+      "name": "today-holidays",
+      "title": "List Today's Holidays",
+      "description": "Allows you to see all holidays happening today for a country and region (if applicable)",
+      "mode": "view",
+      "subtitle": "Upcoming Holidays"
+    },
+    {
+      "name": "pinned-locations",
+      "title": "List Pinned Location Holidays",
+      "description": "Allows you to see all upcoming holidays happening in your pinned locations",
+      "mode": "view",
+      "subtitle": "Upcoming Holidays"
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.35.2",
-    "country-locale-map": "^1.8.0",
-    "moment": "^2.29.3",
-    "node-fetch": "^3.2.4",
-    "swr": "^1.3.0"
```

Comment:
`node-fetch` and `swr` dependencies shouldn't be removed, they are still used in `api.ts`.